From 65aa786d3f6282da02d35b600e82408b47e0a3fe Mon Sep 17 00:00:00 2001
From: Waldemar Brodkorb <wbx@openadk.org>
Date: Mon, 9 Mar 2015 01:48:19 -0500
Subject: [PATCH 20/20] remove NAN selection, better do it in your build
 system

With older gcc than 4.9.x, this breaks the build.
These specific compiler flags belongs better to your
favorite build system.

Signed-off-by: Waldemar Brodkorb <wbx@openadk.org>
---
 Rules.mak                 |    4 ----
 extra/Configs/Config.mips |   15 ---------------
 2 files changed, 19 deletions(-)

diff --git a/Rules.mak b/Rules.mak
index 3b074a8..5217004 100644
--- a/Rules.mak
+++ b/Rules.mak
@@ -404,10 +404,6 @@ ifeq ($(TARGET_ARCH),mips)
 	CPU_CFLAGS-$(CONFIG_MIPS_N64_ABI)+=-mabi=64
 	CPU_CFLAGS-$(CONFIG_MIPS_O32_ABI)+=-mabi=32
 	CPU_CFLAGS-$(CONFIG_MIPS_N32_ABI)+=-mabi=n32
-
-	CPU_CFLAGS-$(CONFIG_MIPS_NAN_LEGACY)+=-mnan=legacy
-	CPU_CFLAGS-$(CONFIG_MIPS_NAN_2008)+=-mnan=2008
-
 	CPU_LDFLAGS-y += $(CPU_CFLAGS)
 endif
 
diff --git a/extra/Configs/Config.mips b/extra/Configs/Config.mips
index 76b10c6..4d30d31 100644
--- a/extra/Configs/Config.mips
+++ b/extra/Configs/Config.mips
@@ -33,21 +33,6 @@ config CONFIG_MIPS_N64_ABI
 endchoice
 
 choice
-	prompt "Target NAN Encoding"
-	default CONFIG_MIPS_NAN_LEGACY
-	help
-	  This is the NAN Encoding you want to use.  Chose either Legacy
-	  or 2008.
-
-config CONFIG_MIPS_NAN_LEGACY
-	bool "LEGACY"
-
-config CONFIG_MIPS_NAN_2008
-	bool "2008"
-
-endchoice
-
-choice
 	prompt "Target Processor Architecture"
 	default CONFIG_MIPS_ISA_1 if CONFIG_MIPS_O32_ABI
 	default CONFIG_MIPS_ISA_3 if CONFIG_MIPS_N32_ABI
-- 
1.7.10.4

